src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -649,7 +649,8 @@ export interface Config {
   urlPrefix: string;
   needsUpgrade: boolean;
   supportEmail: string;
-  user: User;
+  // ?: to work around tests
+  user?: User;
```

Comment:
We should fix or ignore in tests rather than changing the types to be conform to the tests.

Reply:
i fixed this but apparently failed to push it. now the workaround is in configStore.tsx which isn't great but IMO there's too many tests to fix and the only thing that is affected is that it falls back to light mode if user isn't defined which i think is fine.

Reply:
basically a bunch of tests do something like `ConfigStore.set('user', {stuff})` and don't set options.theme, and i'm not sure it's really worth making them manually set light theme when we want that to be the default anyways. i've fixed the type to be non-optional and just work around it in `getTheme`. thank you for the feedback.